src/dids/OracleResolveDriver.ts

```diff
@@ -0,0 +1,134 @@
+import type {DidResolver, DidResolutionResult} from '@credo-ts/core'
+import * as fs from 'fs';
+
+const axios = require("axios");
```

Comment:
Should stick to a single way of importing
```suggestion
import axios from 'axios';
```

Reply:
Good smell catch, will change in next commit push